z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -224,447 +208,107 @@ $ openstack project list
 
-Eventually, the `wait-for` command should end with:
+Others are **optional**, you can enable them and update value if you need more specified settings.
 
-```plaintext
-INFO It is now safe to remove the bootstrap resources
-```
-
-### Access the OpenShift API
-
-You can use the `oc` or `kubectl` commands to talk to the OpenShift API. The admin credentials are in `auth/kubeconfig`:
+> **Please note**: If you want to use a bastion server, you need to set correct value for bastion proprties.
 
-```sh
-$ export KUBECONFIG="$PWD/auth/kubeconfig"
-$ ./oc get nodes
-$ ./oc get pods -A
-```
+| Property| <div style="width:220px">Default</div> | Description                           |
+| --------------------------------------- | ------------------------------------- |:-----|
+| `ansible_ssh_host` | \<linux server ip addr\> | 'x.x.x.x'<br> **required** when use bastion server, give the IP address of bastion server.
+| `bastion_private_ip_address` | \<bastion ip addr\>      |IP address of your bastion node<br>**required** when use bastion server, give the IP address of bastion server.
+| `cluster_subnet_range` |\<cluster subnet range\>       |The queries from IPs in `cluster_subnet_range` will be allowed, we suggest setting the same as `os_subnet_range`  <br>**required** when use bastion server
+| `dns_forwarder` | \<upstream DNS ip addr\> |For nameserver where requests should be forwarded for resolution.<br>**required** when use bastion server
+| `allocation_pool_start` |\<ip range start\> |'x.x.x.x'
+| `allocation_pool_end` |\<ip range end\> |'x.x.x.x'
+| `os_bootstrap_ip` | \<bootstrap ip addr\> |'x.x.x.x'
+| `os_master_ip` | \<master ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x]'
+| `os_infra_ip` |\<infra ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x]'
 
-**NOTE**: Only the API will be up at this point. The OpenShift UI will run on the compute nodes.
 
-### Delete the Bootstrap Resources
+## Create Cluster
 
-```sh
-$ ansible-playbook -i inventory.yaml down-bootstrap.yaml
-```
+- `$ ansible-playbook -i inventory.yaml 01-preparation.yaml`
 
-The teardown playbook deletes the bootstrap port, server and floating IP address.
+> Skip to run bastion.yaml if you want to use your existing DNS/HAProxy or external DNS/HAProxy, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update it.
 
-If you haven't done so already, you should also disable the bootstrap Ignition URL.
+- `ansible-playbook -i inventory.yaml bastion.yaml`
+- `ansible-playbook -i inventory.yaml 02-create-cluster-control.yaml`
+- `ansible-playbook -i inventory.yaml 03-create-cluster-compute.yaml`
 
-### OpenShift Compute Nodes
+After above steps, you will get one ready OpenShift Platform Cluster on ICIC infrastructure.
 
-```sh
-$ ansible-playbook -i inventory.yaml configure-compute-nodes.yaml
-```
+## Something is wrong? clean up environment
```

Comment:
This section is subsection of ## Create Cluster, this is different from below ##Destroy Cluster section. 
Make the title smaller than ##Create Cluster . For example:
## Something is wrong? 
### Any above steps failed ,you need clean up environment and then rerun the installation steps.